docs/build-with-0g/compute-network/sdk.md

```diff
@@ -218,69 +175,41 @@ const completion = await openai.chat.completions.create(
-
-**Note**: Generated `headers` are valid for a single use only and cannot be reused.
+### Response Processing
 
-#### 5.4 Process Responses
+This function is used to verify the response and settle the fee. If it is a verifiable service, it will return whether the response is valid.
 
 ```typescript
-/**
- * 'processResponse' is used after the user successfully obtains a response from the provider service.
- *
- * It will settle the fee for the response content. Additionally, if the service is verifiable,
- * input the chat ID from the response and 'processResponse' will determine the validity of the
- * returned content by checking the provider service's response and corresponding signature associated
- * with the chat ID.
- *
- * @param {string} providerAddress - The address of the provider.
- *
- * @param {string} serviceName - The name of the service.
- *
- * @param {string} content - The main content returned by the service. For example, in the case of a
- * chatbot service, it would be the response text.
- *
- * @param {string} chatID - Only for verifiable services. You can provide the chat ID obtained from the
- * response to automatically download the response signature. The function will verify the reliability
- * of the response using the service's signing address.
- *
- * @returns A boolean value. True indicates the returned content is valid, otherwise it is invalid.
- *
- * @throws An error if any issues occur during the processing of the response.
- */
 const valid = await broker.processResponse(
   providerAddress,
   serviceName,
   content,
-  chatID
+  chatID // Optional: Only for verifiable services
 );
 ```
 
-#### 5.5 Settle Fees Manually
+### Manual Fee Settlement
+
+If `processResponse` fails, you can settle fees manually by calling `settleFee` function.
 
 ```typescript
-/**
- * 'settleFee' is used to settle the fee for the provider service.
- *
- * Normally, the fee for each request will be automatically settled in 'processResponse'.
- * However, if 'processResponse' fails due to network issues or other reasons,
- * you can manually call settleFee to settle the fee.
- *
- * @param {string} providerAddress - The address of the provider.
- *
- * @param {string} serviceName - The name of the service.
- *
- * @param {number} fee - The fee to be settled. The unit is A0GI.
- *
- * @returns A promise that resolves when the fee settlement is successful.
- *
- * @throws An error if any issues occur during the fee settlement process.
- */
 await broker.settleFee(providerAddress, serviceName, fee);
 ```
 
-By following these steps, you can effectively integrate provider services into your applications.
-For more information, please [download the example](./data/ts-sdk-example.ts).
+### Helper Functions
+
+#### Get Account Details
+This function is used to get account details (like balance) on the given provider address.
+```typescript
+const accountDetails = await broker.getAccount(providerAddress);
+```
+
+#### Request Refund
```

Comment:
Should we add a note that this is still WIP and has not been exposed yet? cc @Ravenyjh

Reply:
Great suggestion, we can include this content: 

"When the 'verifiability' attribute of a service is not empty, the service can be verified on the client side. We currently support 'TEEML' (with 'OPML' and 'ZKML' anticipated in the future). 'TEEML' allows each response to be verified as executed in a Trusted Execution Environment using the 'processResponse' function, as detailed here: [processResponse](https://github.com/0glabs/0g-serving-user-broker/blob/f99b523d85daa3be4e2dabac2caa31c2feba4b89/src.ts/broker/response.ts#L78)."

Reply:
agree